schemas/context/implementation.schema.json

```diff
@@ -0,0 +1,35 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/implementation",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Implementation",
+  "type": "object",
+  "meta:extensible": true,
+  "description": "Implementation of an API.",
```

Comment:
This description is insufficient. The term "implementation" is used in a very different way in Adobe Digital Marketing from the rest of the world and "implementation of an API" does not clarify, but rather confuses the issue.

In Adobe Digital Marketing, an Implementation refers to either:
1. client-specific code that implements data collection or targeting using an online service that is running on a consumer's device
2. the SDK or tooling used to enable (1)
3. all of the above

In the rest of the world, "Implementation of an API" means a specific service that conforms to an API, typically describing the server-side, rather than the client side. 

This needs to be clarified – and verified (what are other marketing technology vendors calling their "implementations" – if they use the same term, it's great. If it's is an Adobeism, not so great). As this is most likely to become a multi-line description, it's better to create an `implementation.descrpiption.md` file.

Reply:
I agree, this is open to different interpretations. While "implementation" is not an adobe-ism as it is commonly used to define using a tag + library to add a feature to a web site, e.g google floodlight from google support "Migrate your Floodlight implementation to the global site tag".

The issue is that this does have another meaning for developers generally.

Some ideas I have considered:
- Implementation Details -> Seems to best fit the pattern as it is describing an implementation
- Tagging Implementation -> too web specific, needs to cover apps
- Code Implementation -> too generic.

I will propose "Implementation Details", I'm open to other recommendations.